### PR TITLE
week 5 manual

### PR DESCRIPTION
**Pull request by npant03**

### CODE REVIEW

File: src/main/java/com/team7419/TalonFuncs.java [hunk 1]
```diff
@@ -20,6 +20,5 @@ public static void setPIDFConstants(int slot, BaseTalon talon, double kP, double
 		talon.config_kI(slot, kI, 0);
 		talon.config_kD(slot, kD, 0);
 		talon.config_kF(slot, kF, 0);
-	}
-
+    }
 }
```
Comment (H3nryM) [inline src/main/java/com/team7419/TalonFuncs.java]: I mean everything else looked fine until I saw this file. I'm not too sure about this change.

File: src/main/java/frc/robot/subsystems/intake/RunLoader.java [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
   @Override
   public void initialize() {
+    if(reversed){coeff = -1;}
+    else{coeff = 1;}
   }
 
   @Override
   public void execute() {
-    loader.setPower(power);
+    loader.setPower(coeff*dashboard.getLoaderCoeff());
```
Comment (H3nryM) [inline src/main/java/frc/robot/subsystems/intake/RunLoader.java]: I don't like this naming convention. Why is coeff for reversing the power and coeff is the dashboard power constant thingy?

Comment (npant03) [reply src/main/java/frc/robot/subsystems/intake/RunLoader.java]: Changed it to negativeSign :)

File: src/main/java/frc/robot/subsystems/shooter/GetToTargetVelocity.java [hunk 3]
```diff
@@ -27,29 +25,37 @@ public void initialize() {
...
 
-      double rawSpeed = dashboard.getRawSpeed();
-      kF = dashboard.getShooterkF();
+      target = dashboard.getRawSpeed();
+      shooter.setkF(shooter.lookUpkF(target));
       
-      shooter.configureOutputs();
-      shooter.setPIDF(kP, kI, kD, kF);
-      shooter.setTargetRawSpeed(rawSpeed);
+      double[] gains = dashboard.getRampingGains();
```
Comment (H3nryM) [inline src/main/java/frc/robot/subsystems/shooter/GetToTargetVelocity.java]: oooh me gusta

File: src/main/java/frc/robot/subsystems/shooter/ShooterSub.java [hunk 4]
```diff
@@ -81,14 +73,38 @@ public void setPIDF(double kP, double kI, double kD, double kF){
         TalonFuncs.setPIDFConstants(0, talon, kP, kI, kD, kF);
     }
 
+    public boolean onTarget(){
+        return Math.abs(this.getCurrentRawSpeed() - rawSpeed) < threshold;
```
Comment (H3nryM) [inline src/main/java/frc/robot/subsystems/shooter/ShooterSub.java]: i dont like this naming convention it is confusing

**Pull request merged**